apps/coscrad-frontend/src/components/resources/terms/term-detail.thumbnail.presenter.tsx

```diff
@@ -1,12 +1,33 @@
-import { ICategorizableDetailQueryResult, ITermViewModel } from '@coscrad/api-interfaces';
-import { TermDetailFullViewPresenter } from './term-detail.full-view.presenter';
+import {
+    ICategorizableDetailQueryResult,
+    ITermViewModel,
+    ResourceType,
+} from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import { MultiplePropertyPresenter, PropertyLabels } from '../../../utils/generic-components';
+import { ResourceDetailThumbnailPresenter } from '../../../utils/generic-components/presenters/detail-views';
 
-export const TermDetailThumbnailPresenter = (
-    termAndActions: ICategorizableDetailQueryResult<ITermViewModel>
-): JSX.Element => (
-    /**
-     * For now, we will use the full-view as the thumbnail, since the full-view
-     * is fairly concise to start with.
-     */
-    <TermDetailFullViewPresenter {...termAndActions} />
-);
+export const TermDetailThumbnailPresenter = ({
```

Comment:
this is almost identical to the full-view except that it uses `ResourceDetailThumbnailPresenter`.  But this gives it the thumbnail look for the connected resources list.